app/src/main/res/layout/item_overdue_list.xml

```diff
@@ -21,84 +21,84 @@
-        android:padding="@dimen/spacing_16">
+        android:paddingBottom="@dimen/spacing_16"
+        android:paddingEnd="@dimen/spacing_16"
+        android:paddingTop="@dimen/spacing_16"
+        tools:ignore="RtlSymmetry">
 
         <TextView
-          android:id="@+id/overdue_patient_name_gender"
+          android:id="@+id/overdue_patient_name_age"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
           android:layout_marginBottom="@dimen/spacing_4"
-          android:textColor="@color/black_opacity_90"
+          android:layout_marginStart="@dimen/spacing_16"
+          android:drawablePadding="@dimen/spacing_16"
+          android:drawableStart="@drawable/ic_patient_female"
+          android:textColor="@color/blue1"
           android:textSize="@dimen/textsize_16"
           tools:text="Anisha Acharya" />
 
-        <TextView
-          android:id="@+id/overdue_patient_bp"
+        <LinearLayout
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
           android:layout_marginBottom="@dimen/spacing_4"
-          android:textColor="@color/black_opacity_50"
-          android:textSize="@dimen/textsize_14"
-          tools:text="175/85, 30 days ago" />
-
-        <LinearLayout
-          android:layout_width="match_parent"
-          android:layout_height="wrap_content"
+          android:layout_marginStart="@dimen/spacing_56"
           android:orientation="horizontal">
 
           <TextView
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginBottom="@dimen/spacing_4"
-            android:text="@string/overduelist_age"
+            android:text="@string/overduelist_last_bp"
             android:textAllCaps="true"
-            android:textColor="@color/black_opacity_50"
+            android:textColor="@color/grey1"
             android:textSize="@dimen/textsize_12" />
 
           <TextView
-            android:id="@+id/overdue_patient_age"
+            android:id="@+id/overdue_patient_bp"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginBottom="@dimen/spacing_4"
-            android:paddingEnd="@dimen/spacing_12"
-            android:paddingStart="@dimen/spacing_4"
-            android:textColor="@color/black_opacity_50"
+            android:layout_marginStart="@dimen/spacing_4"
+            android:textColor="@color/grey1"
             android:textSize="@dimen/textsize_14"
-            tools:text="11" />
-
-          <TextView
-            android:id="@+id/overdue_patient_phoneNo"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content" />
-
+            tools:text="175/85, 30 days ago" />
         </LinearLayout>
 
+        <org.simple.clinic.widgets.TintableCompoundDrawableTextView
+          android:id="@+id/overdue_patient_phoneNo"
```

Comment:
`phoneNo` should be `phone_number` -- we refrain from using acronyms and camel casing in XML